scripts/wildcard-domain-processor/top-tld.ts

```diff
@@ -95,15 +97,6 @@ export const TOP_LEVEL_DOMAIN_LIST = [
     'go.id',
     'go.th',
     'gob.mx',
-    'gov',
-    'gov.bd',
-    'gov.co',
-    'gov.in',
-    'gov.my',
-    'gov.np',
-    'gov.tr',
-    'gov.ua',
-    'gov.vn',
```

Comment:
Any reason why these TLDs have been removed?

Reply:
They count as registered domain and added to almost all wildcards

Reply:
@Alex-302 maybe leave just `gov` ?

Reply:
Can't remember examples, where `gov` domains are required.